helm/kiam/CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Helm Chart Changelog
+## 3.0.0
+4 September 2019
+
+Notable Changes:
+* [#292](https://github.com/uswitch/kiam/pull/292) The `extraEnv` parameters in `values.yaml` have been changed to support an array of options.
```

Comment:
I think you should make clear that this is a breaking change and what you need to do to fix your stuff if you are using extraEnv